docs/RFCS/transaction_cancellation.md

```diff
@@ -0,0 +1,51 @@
+- Feature Name: Transaction Cancellation
+- Status: completed (PR [#17252](https://github.com/cockroachdb/cockroach/pull/17252))
+- Start Date: 2017-07-30
+- Author: Julian Gilyadov
+- RFC PR: [#17252](https://github.com/cockroachdb/cockroach/pull/17252)
+- Cockroach Issue: None
+
+# Motivation
+
+It is similar to the motivation behind query cancellation, currently it is possible to monitor sessions and queries, and cancelling queries.  
+Extending this functionality with transaction cancellation would be a logical next step.  
+It would be useful to cancel idle transactions for example.  
```

Comment:
Why is this useful? I'm late to this party, but I'm having a hard time imagining a scenario where cancelling transactions (rather than queries, sessions) is useful in CRDB.